pkg/networkservice/chains/nsmgr/unix_test.go

```diff
@@ -84,6 +85,7 @@ func Test_Local_NoURLUsecase(t *testing.T) {
 }
 
 func Test_MultiForwarderSendfd(t *testing.T) {
+	t.Skip("https://github.com/networkservicemesh/sdk/issues/1118")
```

Comment:
Note: This is disabled because with p2mp rework forwarder should recv fd from the nsmgr. Currently, it is not working in these unit tests by https://github.com/networkservicemesh/sdk/issues/1118 but these scenarios work well in integration tests.